src/modules/FSXAProxyApi.ts

```diff
@@ -138,27 +138,16 @@ export class FSXAProxyApi implements FSXAApi {
     if (pagesize < 1) {
       this._logger.warn(
         'fetchByFilter',
-        `Given pagesize: ${pagesize} was smaller than 1, pagesize will be set to 1`
+        'pagesize must be greater then zero! Using fallback of 30.'
       )
       pagesize = 1
```

Comment:
Please adjust this pagesize fallback accordingly (and fix typos of warnings).

Reply:
Thanks, fixed.